### PR TITLE
Update README and scipy_proc.json for 2020

### LINKED ISSUE #525: Update README and scipy_proc.json for 2020 conference
The dates of the conference and deadlines will need to be updated everywhere. In scipy_proc.json, we can update the conference chair names and affiliations, the conference titles, and the volume number. Everything else will need to wait until after the conference.

### PR DESCRIPTION
**Pull request by mepa**
Updates dates, names, and volume number for 2020 conference.

~Note that some names and affiliations will need to be filled and/or updated sometime closer to the conference dates.~

Updates organizer names and affiliations to match https://www.scipy2020.scipy.org/organisers.

Closes #525

### CODE REVIEW

File: scipy_proc.json [hunk 1]
```diff
@@ -19,38 +19,38 @@
...
+    "year": "2020"
   },
   "organization": [
     {
       "name": "Conference Chairs",
       "members": [
         {
-          "org": "Arizona State University",
-          "name": "Serge Rey"
+          "org": "",
```
Comment (deniederhut) [inline scipy_proc.json]: You can find the organizers' affiliations here: https://www.scipy2020.scipy.org/organisers

File: scipy_proc.json [hunk 2]
```diff
@@ -117,17 +129,17 @@
...
           "org": "Argonne National Laboratory",
           "name": "Scott Collis"
         },
         {
           "org": "Novartis Institutes for Biomedical Research",
           "name": "Eric Ma"
+        },
+        {
+          "org": "",
+          "name": ""
```
Comment (deniederhut) [inline scipy_proc.json]: Are we missing Nadia here?

File: scipy_proc.json [hunk 3]
```diff
@@ -154,45 +166,49 @@
         {
           "org": "University Corporation for Atmospheric Research",
           "name": "Ryan May"
-        },
-        {
```
Comment (deniederhut) [inline scipy_proc.json:158]: Are we missing Juan and Dharhas?

File: scipy_proc.json [hunk 4]
```diff
@@ -154,45 +166,49 @@
...
         {
-          "org": "University of Illinois",
-          "name": "Nathan Goldbaum"
+          "org": "IBM Research Africa",
+          "name": "Celia Cintas"
         },
         {
           "org": "Federal University of Santa Catarina",
-          "name": "Melissa Mendonca"
+          "name": "Melissa Weber Mendonca"
```
Comment (deniederhut) [inline scipy_proc.json:189]: 👍 thanks for catching this!

File: scipy_proc.json [hunk 5]
```diff
@@ -219,107 +235,18 @@
...
+        {
+          "org": "Enthought, Inc.",
+          "name": "Kristen Leiser"
         }
       ]
     },
     {
       "name": "Proceedings Reviewers",
       "members": [
         {
```
Comment (deniederhut) [inline scipy_proc.json:256]: Hm... we might need this to be an empty list for now. I'm worried that a list with an empty object might break the builds with a KeyError somewhere.

**Comment by deniederhut** [comment]
Thanks for taking a stab at this! I've left a few suggestions for changes, but I think one piece of information you might be missing (because I did not think to tell you 😬) is that the organizers are also listed on the main conference page -- https://www.scipy2020.scipy.org/organisers

File: scipy_proc.json (hunk 5 above)
Comment (mepa) [reply scipy_proc.json:256]: The dict isn't empty (please see below), so I think it is fine to leave as is?

File: scipy_proc.json [hunk 6]
```diff
@@ -219,107 +235,18 @@
...
-        },
-        {
-          "name": "Tania Lorido Botran"
-        },
-        {
-          "name": "Yingwei Yu"
-        },
-        {
-          "name": "Zach Golkhou"
+          "name": ""
```
Comment (mepa) [inline scipy_proc.json:257]: @deniederhut, the list contains a dict with `key: value` of `"name": ""` (way down here in the diff!)

**Comment by mepa** [comment]
Thanks, @deniederhut! I updated the organizer names and affiliations from the main conference page (I was referring to a Google doc and somehow didn't think to check the page :-D)

I left the Proceedings Reviewers section same since I think it should work fine as is for the build (I haven't tested explicitly, though I should figure out how to test that). But please lmk if you think otherwise.

File: scipy_proc.json (hunk 5 above)
Comment (deniederhut) [reply scipy_proc.json:256]: Hmm yeah I'm not sure what I was looking at

**Review by deniederhut — APPROVED** [review]
Thanks Meghann!

**Pull request merged**